Formula/djl-serving.rb

```diff
@@ -0,0 +1,64 @@
+class DjlServing < Formula
+  desc "This module contains an universal model serving implementation"
+  homepage "https://github.com/awslabs/djl/tree/master/serving"
+  url "https://djl-ai.s3.amazonaws.com/publish/djl-serving/serving-0.11.0.tar"
+  sha256 "7d06748e5084c93cd9afbb5a83a89a4efde1bcb72b2c7921a3f7ab0352d5e852"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    # Install files
+    rm_rf Dir["bin/*.bat"]
+    mv "bin/serving", "bin/djl-serving"
+    libexec.install Dir["*"]
+    (bin/"djl-serving").write_env_script "#{libexec}/bin/djl-serving", Language::Java.overridable_java_home_env
+  end
+
+  plist_options manual: "djl-serving"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Disabled</key>
+          <false/>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{bin}/djl-serving</string>
+            <string>run</string>
+          </array>
+          <key>KeepAlive</key>
+          <true/>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    assert_predicate testpath/"#{bin}/djl-serving", :exist?
+    assert_predicate testpath/"#{libexec}/lib/serving-0.11.0.jar", :exist?
```

Comment:
We don't really need to check if brew can install files. We have unittests for that.